lib/ccf/one_subject_job_submitter.py

```diff
@@ -569,13 +569,16 @@ def create_mark_no_longer_running_script(self):
         script.write('source ' + self._get_xnat_pbs_setup_script_path() + ' ' + self._get_db_name() + os.linesep)
         script.write(os.linesep)
         script.write(self.mark_running_status_program_path + ' \\' + os.linesep)
-        script.write('  --project=' + self.project + ' \\' + os.linesep)
-        script.write('  --subject=' + self.subject + ' \\' + os.linesep)
-        script.write('  --classifier=' + self.classifier + ' \\' + os.linesep)
-
+        script.write('  --user="' + self.username + '" \\' + os.linesep)
+        script.write('  --password="' + self.password + '" \\' + os.linesep)
+        script.write('  --server="' + str_utils.get_server_name(self.put_server) + '" \\' + os.linesep)
+        script.write('  --project="' + self.project + '" \\' + os.linesep)
+        script.write('  --subject="' + self.subject + '" \\' + os.linesep)
+        script.write('  --session="' + self.session + '" \\' + os.linesep)
         if self.scan:
-            script.write('  --scan=' + self.scan + ' \\' + os.linesep)
-        
+            script.write('  --scan="' + self.scan + '" \\' + os.linesep)
+        script.write('  --resource="' + 'RunningStatus' + '" \\' + os.linesep)
+        script.write('  --reason="' + 'RunningStatus' + '" \\' + os.linesep)
```

Comment:
Of course, this `--reason=` parameter will be removed when the other changes mentioned are made.

Reply:
removed reason line, and updated XNAT_MARK_RUNNING_STATUS